src/arrays.js

```diff
@@ -1,61 +1,59 @@
 const getNthElement = (index, array) => {
-  // your code here
+  if (index >= array.length)
+    { 
+      return array[index -array.length];
+    }
+ 
+      return array[index];
+  
 };
 
 const arrayToCSVString = (array) => {
-  // your code here
+  return array.join();
 };
 
 const csvStringToArray = (string) => {
-  // your code here
+  return string.split(',');
 };
 
-const addToArray = (element, array) => {
-  // your code here
-};
+const addToArray = (element, array) => { array.push(element) };
 
-const addToArray2 = (element, array) => {
-  // your code here
-};
+const addToArray2 = (element, array) =>  array.concat(element);
 
-const removeNthElement = (index, array) => {
-  // your code here
-};
+const removeNthElement = (index, array) =>  array.splice(index, 1);
+   
 
-const numbersToStrings = (numbers) => {
-  // your code here
-};
+const numbersToStrings = (numbers) => numbers.map(num => String(num));
 
-const uppercaseWordsInArray = (strings) => {
-  // your code here
-};
+const uppercaseWordsInArray = (strings) =>  strings.map(element => element.toUpperCase());
 
-const reverseWordsInArray = (strings) => {
-  // your code here
+const reverseWordsInArray = (strings) => 
+{
+  const reverseString = (word) => word.split("").reverse().join('');
```

Comment:
Either double quotes or single quotes is fine, but choose one and try to stick to it.